ProjectSettings/ProjectSettings.asset

```diff
@@ -619,32 +598,34 @@ PlayerSettings:
   webGLTemplate: APPLICATION:Default
   webGLAnalyzeBuildSize: 0
   webGLUseEmbeddedResources: 0
-  webGLUseWasm: 0
   webGLCompressionFormat: 1
+  webGLLinkerTarget: 0
   scriptingDefineSymbols:
     14: 
   platformArchitecture: {}
   scriptingBackend:
     Metro: 2
     Standalone: 0
     WebPlayer: 0
+  il2cppCompilerConfiguration: {}
   incrementalIl2cppBuild: {}
+  allowUnsafeCode: 0
   additionalIl2CppArgs: 
-  scriptingRuntimeVersion: 0
+  scriptingRuntimeVersion: 1
   apiCompatibilityLevelPerPlatform:
     Metro: 3
   m_RenderingPath: 1
   m_MobileRenderingPath: 1
-  metroPackageName: MixedRealityToolkit-Unity
-  metroPackageVersion: 2017.3.1.0
+  metroPackageName: Microsoft.MixedReality.Toolkit
+  metroPackageVersion: 1.0.0.0
```

Comment:
What was the first version we agreed on?

Reply:
Darn it, in one update I changed it to 2018.1.0 Which seems to have not stuck. 
DO we want to change this again for this PR?

Reply:
It's not important

Reply:
Not important now. I expect we will use yymm.release.patch or yyyy.mm.patch when we hit alpha.